src/main/java/duke/Launcher.java

```diff
@@ -0,0 +1,12 @@
+package duke;
+
+import javafx.application.Application;
+
+/**
+ * A launcher class to workaround classpath issues.
+ */
+public class Launcher {
+    public static void main(String[] args) {
+        Application.launch(Main.class, args);
+    }
+}
```

Comment:
I think you might want to delete trailing whitespace and add newline at end of file